assets/js/index.js

```diff
@@ -3,8 +3,16 @@
 
   function setupSelect2For(name) {
     const perPage = 10;
+    const input = $('select[name="' + name + '"]');
+    const form = input.parents('form');
 
-    $('select[name="' + name + '"]').select2({
+    function _submitFormIfPressedEnter(e) {
+      if (e.keyCode === 13) {
```

Comment:
well, we could hack it even more and sticking a `hide()` in there :skull_and_crossbones: 

...or we could leave it this way.